compact/far_backward/lumi_design_1/exitWindow.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lccdd>
+
+  <!-- Luminosity exit window -->
+
+  <define>
+    <constant name="gap" value="1*mm"/>
+  </define>
+
+<<<<<<< HEAD
```

Comment:
There are conflict tags here.

Reply:
Just pushed the fix.  What tool could I use to check for conflicting tags within other xmls?

Reply:
oops.  Now I saw what you meant.  pushed another fix.